app/app.go

```diff
@@ -441,6 +441,7 @@ func New(
 		app.BankKeeper,
 		app.StakingKeeper,
 		app.DistrKeeper,
+		authtypes.FeeCollectorName,
```

Comment:
Suggest to use `authtypes.NewModuleAddress(authtypes.FeeCollectorName).String(),` here instead. This is similar to how other keepers that accept module accounts do it

Reply:
The reason of passing `authtypes.FeeCollectorName` as the module name is that this pattern has been used by `MintKeeper` and `DistrKeeper`. Also if we use the actual account address by passing `authtypes.NewModuleAddress(authtypes.FeeCollectorName).String()`, we need to change `k.bankKeeper.SendCoinsFromModuleToModule` to `k.bankKeeper.SendCoinsFromModuleToAccount`.

I can change this if you prefer this pattern